front/src/applications/stdcm/utils/formatStdcmConfV2.ts

```diff
@@ -119,32 +97,13 @@ export const checkStdcmConf = (
     );
   }
 
-  const startTime = dateTimeToIso(`${originDate}T${originTime}`);
-
-  let startDateTime: Date | null;
+  const firstArrival = pathSteps[0]?.arrival;
+  const lastArrival = last(pathSteps)?.arrival;
+  const isDepartureScheduled = pathSteps[0]?.arrivalType === 'preciseTime';
```

Comment:
```suggestion
  const origin = pathSteps.at(0);
  const destination = pathSteps.at(-1);
  
  const firstArrival = origin?.arrival;
  const lastArrival = destination?.arrival;
  const isDepartureScheduled = origin?.arrivalType === 'preciseTime';
```
You can define `origin` and `destination` at the beginning of the function